CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing to GrimoireLab Sorting Hat
+
+These are some general guidelines and information related to how we contribute to 
+GrimoireLab. You can read about it from the [CONTRIBUTING.md](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md).
+
+## Changelog Entries
+
+Some of your contributions will require a changelog entry which explains 
+the motivation of the change. These changelog entries should be written 
+to the `releases/unreleased/` directory. These entries would be included 
+in the release notes to explain users and developers about the new features 
+or bugs fixed in the software. This is an example of changelog entry:
+
+```
+title: 'Fix bug casting spells on magicians'
+category: fixed
+author: John Smith <jsmith@example.com>
+issue: 666
+notes: >
+    The bug was making impossible to cast a spell on
+    a magician.
+```
+
+Changelog entries will be written to explain *what* was changed and *why*, 
+not *how*. Take into account not everybody is a developer and they are 
+meant to reach a wider audience.
+
+### What warrants a changelog entry
+
+The changelog entry is **required** for
+- Any change in the code that directly affects the GrimoireLab users
+- Bug fixes
+- New updates
+- Performance improvements
+
+The changelog entry is **not required** for
+- Any docs-only (e.g., README.md) changes
+- Any developer-facing change (e.g., test suite changes)
+
+### Writing changelog entries
+
+```
+title: 'Fix bug casting spells on magicians'
+category: fixed
+author: John Smith <jsmith@example.com>
+issue: 666
+notes: >
+    The bug was making impossible to cast a spell on
+    a magician.
+```
+
+The `issue` value is a reference to the issue, if any, that is targeted with 
+this change, and the `author` key (format: `Your Name <YourName@example.org>`) 
+is used to give attribution to community contributors. **Both are optional.** 
+The `category` field maps the category of the change, valid options are: 
+added, fixed, changed, deprecated, removed, security, performance, other. 
+**Category field is mandatory.** The `notes` field should have a description 
+explaining the changes in the code. Remember you can write blocks of text using 
+the `>` character at the beginning of each block. See the above given example.
```

Comment:
I'd start explaining the title and the category fields. Then move to the other ones. A paragraph for each field.